analysis/Scripts/Efficiencies/efficiencies.py

```diff
@@ -137,32 +158,40 @@ def main(argv):
 					run = 2
 					cuts = Imports.getDataCuts(run)
 					turbo = "lcplus_Hlt1TrackMVADecision_TOS==1"
-				
+
 				#WHAT DO I NEED FOR TRIGGER CUTS?!!
- 
+
 				Lc_MC_tree = TChain("tuple_Lc2pKpi/DecayTree") # !!! QUESTION : NOT BETTER ISTEAD OF CHAIN; JUST GETENTRIES FROM EACH ONE BY ONE, ONCE WITHOUT CUT AND ONCE WITH?
```

Comment:
How long does it take to make the TChain, get entries, then cut and get entries? If it's a limiting factor for your time, we could consider alternatives.

Reply:
I think it's actually ok, it doesn't take huge lengths of time